[NU-1071] Processing mode related changes in the documentation (#5747)
TouK/nussknacker: docs/installation_configuration_guide/ScenarioDeploymentConfiguration.md

sidebar_position: 3
---

# Scenario Deployment configuration

In order to deploy scenario on the given [Engine](/about/engine), you need to configure the deployment.

> For the scenario's deployment is responsible Designer's extension called [Deployment Manager](/about/GLOSSARY#deployment-manager).

Reviewer: ```suggestion
Deployment of a scenario is managed by Designer's extension called [Deployment Manager](/about/GLOSSARY#deployment-manager).
```

> To enable given [Deployment Manager](/about/GLOSSARY#deployment-manager) you need to place their jar package in the Designer's classpath.

Reviewer: ```suggestion
To enable given [Deployment Manager](/about/GLOSSARY#deployment-manager) you need to place its jar package in the Designer's classpath.
```

Nussknacker is distributed with three default [Deployment Managers](/about/GLOSSARY#deployment-manager) (`flinkStreaming`, `lite-k8s`, `lite-embedded`). Their jars are located in the `managers`

> directory. Parameters available in the deployment configuration, depends on which [Deployment Manager](/about/GLOSSARY#deployment-manager) you've selected.

Reviewer: ```suggestion
directory. Your deployment configuration should match the [Deployment Manager](/about/GLOSSARY#deployment-manager) you've selected.
```
Don't know if that's what you meant. I read it as "If you set type = "flinkStreaming" you configure that specific DM and you should provide values for it specifically"

Author: changed

> Section with `deploymentConfig` need to be placed in the correct place in the Designer's configuration. Check [configuration areas](./Common.md#configuration-areas) to understand the structure of the configuration.

Reviewer: ```suggestion
Section with `deploymentConfig` needs to be placed in the correct place in the Designer's configuration. Check [configuration areas](./Common.md#configuration-areas) to understand the structure of the configuration.
```
